File: C#/CommandEventExample/PHP/MyApp/Events/FooedHandler.php
<?php

declare(strict_types=1);


namespace MyApp\Events;

use MyLibrary\Event\Handler;
use MyLibrary\IEvent;

final class FooedHandler extends Handler
{

    public function Handle(IEvent $event): void
    {
        echo "Fooed event fired with payload: \"{$event->Payload()->Value()}\"";
    }
}

File: Java/CommandsEvents/src/MyApp/Events/FooedHandler.java
package MyApp.Events;

import MyLibrary.Event.Handler;

public final class FooedHandler extends Handler<Fooed>
{
    @Override
    public void Handle(Fooed fooed)
    {
        System.out.printf("Fooed event fired with payload: \"%s\"%n", fooed.Payload().Value());
    }
}
